### DISCUSSION TITLE
Future plans for this project

### DISCUSSION THREAD

**Discussion by lefticus** [lefticus/cpp_starter_project#158] [General]
We've had a lot of movement lately clearing out old PRs, cleaning up issues, and fixing the dashboard.

I think we should take time now to slow down, and decide what issues / PRs need to get closed to get the project back to what we consider to be a "stable" state before adding any new features.

I'll create a new milestone that we can start tagging things for, and we can discuss how to move forward here as necessary.

Re: @ddalcino and @aminya

**Comment by lefticus** [comment]
I created "1.0" milestone, please review and update/comment if there should be any changes

https://github.com/cpp-best-practices/cpp_starter_project/milestone/1

**Reply by aminya** [reply]
This is great! Thanks.

**Comment by ddalcino** [comment]
I am tempted to add #159 to this list, since I believe that it's important. I think it would be presumptuous to unilaterally add my own PR to the milestone though, so I won't do it.
